### DISCUSSION TITLE
Docker installation problem

### DISCUSSION THREAD

**Discussion by xansone** [mediacms-io/mediacms#876] [Q&A]
Good afternoon,
Im a newbie in this stuff and i didnt want to bother you guys but i cant find solution to my problem. 
I decided to choose docker installation, because i've got already running services like flarum, tsnranksystem etc.
I modyfied a little docker-compose because i had troubles with postgres with error role "root" not define and i changed ports to 8080:8080, installation starts but its stack on redis moment.

Attaching to mediacms_db_1, mediacms_redis_1, mediacms_celery_beat_1, mediacms_migrations_1, mediacms_celery_worker_1, mediacms_web_1
celery_worker_1  | Checking for script in deploy/docker/prestart.sh
celery_worker_1  | Running script deploy/docker/prestart.sh
celery_worker_1  | Enabling celery-short task worker
celery_worker_1  | Enabling celery-long task worker
celery_worker_1  | Starting server using supervisord...
celery_worker_1  | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
celery_worker_1  |   self.warnings.warn(
celery_worker_1  | 2023-09-08 08:47:28,171 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
celery_worker_1  | 2023-09-08 08:47:28,172 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_long.conf" during parsing
celery_worker_1  | 2023-09-08 08:47:28,172 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_short.conf" during parsing
celery_worker_1  | 2023-09-08 08:47:28,172 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
celery_worker_1  | 2023-09-08 08:47:28,179 INFO RPC interface 'supervisor' initialized
celery_worker_1  | 2023-09-08 08:47:28,179 CRIT Server 'unix_http_server' running without any HTTP authentication checking
celery_worker_1  | 2023-09-08 08:47:28,179 INFO supervisord started with pid 1
celery_worker_1  | 2023-09-08 08:47:29,183 INFO spawned: 'celery_short' with pid 54
celery_worker_1  | 2023-09-08 08:47:29,187 INFO spawned: 'celery_long' with pid 55
celery_worker_1  | celery multi v5.3.1 (emerald-rush)
celery_worker_1  | > Starting nodes...
celery_worker_1  |      > long1@5eb28b7472c8: 2023-09-08 08:47:30,084 INFO success: celery_short entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_worker_1  | 2023-09-08 08:47:30,084 INFO success: celery_long entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_worker_1  | celery multi v5.3.1 (emerald-rush)
celery_worker_1  | > Starting nodes...
celery_worker_1  |      > short1@5eb28b7472c8: OK
celery_worker_1  |      > short2@5eb28b7472c8: OK2023-09-08 08:47:30,910 INFO reaped unknown pid 58 (exit status 0)
celery_worker_1  |
celery_worker_1  | OK
celery_worker_1  | 2023-09-08 08:47:31,737 INFO exited: celery_long (exit status 0; expected)
celery_worker_1  | 2023-09-08 08:47:31,737 INFO reaped unknown pid 61 (exit status 0)
celery_worker_1  | 2023-09-08 08:47:31,737 INFO reaped unknown pid 63 (exit status 0)
celery_worker_1  | 2023-09-08 08:47:32,741 INFO exited: celery_short (exit status 0; expected)
celery_beat_1    | Checking for script in deploy/docker/prestart.sh
celery_beat_1    | Running script deploy/docker/prestart.sh
celery_beat_1    | Enabling celery-beat scheduling server
celery_beat_1    | Starting server using supervisord...
celery_beat_1    | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
celery_beat_1    |   self.warnings.warn(
celery_beat_1    | 2023-09-08 07:42:42,637 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
celery_beat_1    | 2023-09-08 07:42:42,638 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_beat.conf" during parsing
celery_beat_1    | 2023-09-08 07:42:42,638 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
celery_beat_1    | 2023-09-08 07:42:42,643 INFO RPC interface 'supervisor' initialized
celery_beat_1    | 2023-09-08 07:42:42,643 CRIT Server 'unix_http_server' running without any HTTP authentication checking
celery_beat_1    | 2023-09-08 07:42:42,645 INFO supervisord started with pid 1
celery_beat_1    | 2023-09-08 07:42:43,649 INFO spawned: 'celery_beat' with pid 53
celery_beat_1    | 2023-09-08 07:42:44,651 INFO success: celery_beat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_beat_1    | celery beat v5.3.1 (emerald-rush) is starting.
celery_beat_1    | __    -    ... __   -        _
celery_beat_1    | LocalTime -> 2023-09-08 08:42:46
celery_beat_1    | Configuration ->
celery_beat_1    |     . broker -> redis://redis:6379/1
celery_beat_1    |     . loader -> celery.loaders.app.AppLoader
celery_beat_1    |     . scheduler -> celery.beat.PersistentScheduler
celery_beat_1    |     . db -> celerybeat-schedule
celery_beat_1    |     . logfile -> /home/mediacms.io/mediacms/logs/celery_beat.log@%INFO
celery_beat_1    |     . maxinterval -> 5.00 minutes (300s)
celery_beat_1    | 2023-09-08 07:46:05,693 WARN received SIGTERM indicating exit request
celery_beat_1    | 2023-09-08 07:46:05,694 INFO waiting for celery_beat to die
celery_beat_1    | 2023-09-08 07:46:07,193 INFO stopped: celery_beat (exit status 0)
celery_beat_1    | Checking for script in deploy/docker/prestart.sh
celery_beat_1    | Running script deploy/docker/prestart.sh
celery_beat_1    | Enabling celery-beat scheduling server
celery_beat_1    | Starting server using supervisord...
celery_beat_1    | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
celery_beat_1    |   self.warnings.warn(
celery_beat_1    | 2023-09-08 07:47:02,660 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
celery_beat_1    | 2023-09-08 07:47:02,660 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_beat.conf" during parsing
celery_beat_1    | 2023-09-08 07:47:02,660 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
celery_beat_1    | 2023-09-08 07:47:02,665 INFO RPC interface 'supervisor' initialized
celery_beat_1    | 2023-09-08 07:47:02,665 CRIT Server 'unix_http_server' running without any HTTP authentication checking
celery_beat_1    | 2023-09-08 07:47:02,665 INFO supervisord started with pid 1
celery_beat_1    | 2023-09-08 07:47:03,669 INFO spawned: 'celery_beat' with pid 38
celery_beat_1    | celery beat v5.3.1 (emerald-rush) is starting.
celery_beat_1    | 2023-09-08 07:47:04,472 INFO success: celery_beat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_beat_1    | __    -    ... __   -        _
celery_beat_1    | LocalTime -> 2023-09-08 08:47:05
celery_beat_1    | Configuration ->
celery_beat_1    |     . broker -> redis://redis:6379/1
celery_beat_1    |     . loader -> celery.loaders.app.AppLoader
celery_beat_1    |     . scheduler -> celery.beat.PersistentScheduler
celery_beat_1    |     . db -> celerybeat-schedule
celery_beat_1    |     . logfile -> /home/mediacms.io/mediacms/logs/celery_beat.log@%INFO
celery_beat_1    |     . maxinterval -> 5.00 minutes (300s)
celery_beat_1    | 2023-09-08 07:50:42,839 WARN received SIGTERM indicating exit request
celery_beat_1    | 2023-09-08 07:50:42,841 INFO waiting for celery_beat to die
celery_beat_1    | 2023-09-08 07:50:44,341 INFO stopped: celery_beat (exit status 0)
celery_beat_1    | Checking for script in deploy/docker/prestart.sh
celery_beat_1    | Running script deploy/docker/prestart.sh
celery_beat_1    | Enabling celery-beat scheduling server
celery_beat_1    | Starting server using supervisord...
celery_beat_1    | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
celery_beat_1    |   self.warnings.warn(
celery_beat_1    | 2023-09-08 07:51:38,119 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
celery_beat_1    | 2023-09-08 07:51:38,119 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_beat.conf" during parsing
celery_beat_1    | 2023-09-08 07:51:38,119 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
celery_beat_1    | 2023-09-08 07:51:38,124 INFO RPC interface 'supervisor' initialized
celery_beat_1    | 2023-09-08 07:51:38,124 CRIT Server 'unix_http_server' running without any HTTP authentication checking
celery_beat_1    | 2023-09-08 07:51:38,124 INFO supervisord started with pid 1
celery_beat_1    | 2023-09-08 07:51:39,128 INFO spawned: 'celery_beat' with pid 39
celery_beat_1    | celery beat v5.3.1 (emerald-rush) is starting.
celery_beat_1    | 2023-09-08 07:51:39,892 INFO success: celery_beat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_beat_1    | __    -    ... __   -        _
celery_beat_1    | LocalTime -> 2023-09-08 08:51:41
celery_beat_1    | Configuration ->
celery_beat_1    |     . broker -> redis://redis:6379/1
celery_beat_1    |     . loader -> celery.loaders.app.AppLoader
celery_beat_1    |     . scheduler -> celery.beat.PersistentScheduler
celery_beat_1    |     . db -> celerybeat-schedule
celery_beat_1    |     . logfile -> /home/mediacms.io/mediacms/logs/celery_beat.log@%INFO
celery_beat_1    |     . maxinterval -> 5.00 minutes (300s)
celery_beat_1    | 2023-09-08 08:35:03,967 WARN received SIGTERM indicating exit request
celery_beat_1    | 2023-09-08 08:35:03,985 INFO waiting for celery_beat to die
celery_beat_1    | 2023-09-08 08:35:04,496 INFO stopped: celery_beat (exit status 0)
celery_beat_1    | Checking for script in deploy/docker/prestart.sh
celery_beat_1    | Running script deploy/docker/prestart.sh
celery_beat_1    | Enabling celery-beat scheduling server
celery_beat_1    | Starting server using supervisord...
celery_beat_1    | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
celery_beat_1    |   self.warnings.warn(
celery_beat_1    | 2023-09-08 08:45:56,839 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
celery_beat_1    | 2023-09-08 08:45:56,840 INFO Included extra file "/etc/supervisor/conf.d/supervisord-celery_beat.conf" during parsing
celery_beat_1    | 2023-09-08 08:45:56,840 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
celery_beat_1    | 2023-09-08 08:45:56,845 INFO RPC interface 'supervisor' initialized
celery_beat_1    | 2023-09-08 08:45:56,845 CRIT Server 'unix_http_server' running without any HTTP authentication checking
celery_beat_1    | 2023-09-08 08:45:56,845 INFO supervisord started with pid 1
celery_beat_1    | 2023-09-08 08:45:57,849 INFO spawned: 'celery_beat' with pid 39
celery_beat_1    | 2023-09-08 08:45:58,851 INFO success: celery_beat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_beat_1    | celery beat v5.3.1 (emerald-rush) is starting.
celery_beat_1    | __    -    ... __   -        _
celery_beat_1    | LocalTime -> 2023-09-08 09:46:00
celery_beat_1    | Configuration ->
celery_beat_1    |     . broker -> redis://redis:6379/1
celery_beat_1    |     . loader -> celery.loaders.app.AppLoader
celery_beat_1    |     . scheduler -> celery.beat.PersistentScheduler
celery_beat_1    |     . db -> celerybeat-schedule
celery_beat_1    |     . logfile -> /home/mediacms.io/mediacms/logs/celery_beat.log@%INFO
celery_beat_1    |     . maxinterval -> 5.00 minutes (300s)
db_1             |
db_1             | PostgreSQL Database directory appears to contain a database; Skipping initialization
db_1             |
db_1             | 2023-09-08 09:48:27.099 BST [1] LOG:  starting PostgreSQL 15.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r4) 12.2.1 20220924, 64-bit
db_1             | 2023-09-08 09:48:27.100 BST [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1             | 2023-09-08 09:48:27.100 BST [1] LOG:  listening on IPv6 address "::", port 5432
db_1             | 2023-09-08 09:48:27.104 BST [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1             | 2023-09-08 09:48:27.108 BST [24] LOG:  database system was shut down at 2023-09-08 09:48:26 BST
db_1             | 2023-09-08 09:48:27.113 BST [1] LOG:  database system is ready to accept connections
web_1            | Checking for script in deploy/docker/prestart.sh
web_1            | Running script deploy/docker/prestart.sh
web_1            | Enabling uwsgi app server
web_1            | Enabling nginx as uwsgi app proxy and media server
web_1            | Starting server using supervisord...
web_1            | /usr/lib/python3/dist-packages/supervisor/options.py:474: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
web_1            |   self.warnings.warn(
web_1            | 2023-09-08 08:49:34,334 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
web_1            | 2023-09-08 08:49:34,334 INFO Included extra file "/etc/supervisor/conf.d/supervisord-debian.conf" during parsing
web_1            | 2023-09-08 08:49:34,334 INFO Included extra file "/etc/supervisor/conf.d/supervisord-nginx.conf" during parsing
web_1            | 2023-09-08 08:49:34,334 INFO Included extra file "/etc/supervisor/conf.d/supervisord-uwsgi.conf" during parsing
web_1            | 2023-09-08 08:49:34,338 INFO RPC interface 'supervisor' initialized
web_1            | 2023-09-08 08:49:34,338 CRIT Server 'unix_http_server' running without any HTTP authentication checking
web_1            | 2023-09-08 08:49:34,338 INFO supervisord started with pid 1
web_1            | 2023-09-08 08:49:35,342 INFO spawned: 'uwsgi' with pid 54
web_1            | 2023-09-08 08:49:35,344 INFO spawned: 'nginx' with pid 55
web_1            | [uWSGI] getting INI configuration from /home/mediacms.io/mediacms/deploy/docker/uwsgi.ini
web_1            | *** Starting uWSGI 2.0.21 (64bit) on [Fri Sep  8 08:49:35 2023] ***
web_1            | 2023-09-08 08:49:35,358 INFO success: uwsgi entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
web_1            | 2023-09-08 08:49:35,358 INFO success: nginx entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
web_1            | compiled with version: 12.2.0 on 17 July 2023 13:56:25
web_1            | os: Linux-5.4.0-152-generic #169-Ubuntu SMP Tue Jun 6 22:23:09 UTC 2023
web_1            | nodename: f2c246ddb856
web_1            | machine: x86_64
web_1            | clock source: unix
web_1            | detected number of CPU cores: 6
web_1            | current working directory: /home/mediacms.io/mediacms
web_1            | detected binary path: /home/mediacms.io/bin/uwsgi
web_1            | !!! no internal routing support, rebuild with pcre support !!!
web_1            | setgid() to 33
web_1            | set additional group 0 (root)
web_1            | setuid() to 33
web_1            | chdir() to /home/mediacms.io/mediacms/
web_1            | your memory page size is 4096 bytes
web_1            | detected max file descriptor number: 1048576
web_1            | lock engine: pthread robust mutexes
web_1            | thunder lock: disabled (you can enable it with --thunder-lock)
web_1            | uwsgi socket 0 bound to TCP address 127.0.0.1:9000 fd 3
web_1            | Python version: 3.11.4 (main, Jul  4 2023, 05:25:16) [GCC 12.2.0]
web_1            | PEP 405 virtualenv detected: /home/mediacms.io
web_1            | Set PythonHome to /home/mediacms.io
web_1            | Python main interpreter initialized at 0x7f2ae868c578
redis_1          | 1:C 08 Sep 2023 07:42:05.454 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
redis_1          | 1:C 08 Sep 2023 07:42:05.454 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1          | 1:C 08 Sep 2023 07:42:05.454 * Redis version=7.2.1, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1          | 1:C 08 Sep 2023 07:42:05.454 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis_1          | 1:M 08 Sep 2023 07:42:05.455 * monotonic clock: POSIX clock_gettime
redis_1          | 1:M 08 Sep 2023 07:42:05.456 * Running mode=standalone, port=6379.
redis_1          | 1:M 08 Sep 2023 07:42:05.456 * Server initialized
redis_1          | 1:M 08 Sep 2023 07:42:05.457 * Ready to accept connections tcp
redis_1          | 1:signal-handler (1694159167) Received SIGTERM scheduling shutdown...
redis_1          | 1:M 08 Sep 2023 07:46:07.638 * User requested shutdown...
redis_1          | 1:M 08 Sep 2023 07:46:07.638 * Saving the final RDB snapshot before exiting.
redis_1          | 1:M 08 Sep 2023 07:46:07.648 * DB saved on disk
redis_1          | 1:M 08 Sep 2023 07:46:07.648 # Redis is now ready to exit, bye bye...
redis_1          | 1:C 08 Sep 2023 07:47:01.522 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1          | 1:C 08 Sep 2023 07:47:01.522 * Redis version=7.2.1, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1          | 1:C 08 Sep 2023 07:47:01.522 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis_1          | 1:M 08 Sep 2023 07:47:01.522 * monotonic clock: POSIX clock_gettime
redis_1          | 1:M 08 Sep 2023 07:47:01.523 * Running mode=standalone, port=6379.
redis_1          | 1:M 08 Sep 2023 07:47:01.524 * Server initialized
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * Loading RDB produced by version 7.2.1
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * RDB age 54 seconds
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * RDB memory usage when created 1.25 Mb
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * Done loading RDB, keys loaded: 4, keys expired: 0.
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * DB loaded from disk: 0.000 seconds
redis_1          | 1:M 08 Sep 2023 07:47:01.525 * Ready to accept connections tcp
redis_1          | 1:signal-handler (1694159444) Received SIGTERM scheduling shutdown...
redis_1          | 1:M 08 Sep 2023 07:50:44.792 * User requested shutdown...
redis_1          | 1:M 08 Sep 2023 07:50:44.792 * Saving the final RDB snapshot before exiting.
redis_1          | 1:M 08 Sep 2023 07:50:44.795 * DB saved on disk
redis_1          | 1:M 08 Sep 2023 07:50:44.795 # Redis is now ready to exit, bye bye...
redis_1          | 1:C 08 Sep 2023 07:51:36.975 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1          | 1:C 08 Sep 2023 07:51:36.975 * Redis version=7.2.1, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1          | 1:C 08 Sep 2023 07:51:36.975 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis_1          | 1:M 08 Sep 2023 07:51:36.975 * monotonic clock: POSIX clock_gettime
redis_1          | 1:M 08 Sep 2023 07:51:36.979 * Running mode=standalone, port=6379.
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * Server initialized
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * Loading RDB produced by version 7.2.1
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * RDB age 52 seconds
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * RDB memory usage when created 1.25 Mb
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * Done loading RDB, keys loaded: 4, keys expired: 0.
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * DB loaded from disk: 0.000 seconds
redis_1          | 1:M 08 Sep 2023 07:51:36.980 * Ready to accept connections tcp
redis_1          | 1:signal-handler (1694162104) Received SIGTERM scheduling shutdown...
redis_1          | 1:M 08 Sep 2023 08:35:04.965 * User requested shutdown...
redis_1          | 1:M 08 Sep 2023 08:35:04.965 * Saving the final RDB snapshot before exiting.
web_1            | python threads support enabled
web_1            | your server socket listen backlog is limited to 100 connections
web_1            | your mercy for graceful operations on workers is 60 seconds
web_1            | mapped 422160 bytes (412 KB) for 4 cores
web_1            | *** Operational MODE: preforking+threaded ***
web_1            | WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x7f2ae868c578 pid: 54 (default app)
web_1            | *** uWSGI is running in multiple interpreter mode ***
web_1            | spawned uWSGI master process (pid: 54)
web_1            | spawned uWSGI worker 1 (pid: 64, cores: 2)
web_1            | spawned uWSGI worker 2 (pid: 66, cores: 2)
web_1            | running "unix_signal:15 gracefully_kill_them_all" (master-start)...
redis_1          | 1:M 08 Sep 2023 08:35:04.968 * DB saved on disk
redis_1          | 1:M 08 Sep 2023 08:35:04.968 # Redis is now ready to exit, bye bye...
redis_1          | 1:C 08 Sep 2023 08:45:54.216 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1          | 1:C 08 Sep 2023 08:45:54.216 * Redis version=7.2.1, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1          | 1:C 08 Sep 2023 08:45:54.216 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis_1          | 1:M 08 Sep 2023 08:45:54.216 * monotonic clock: POSIX clock_gettime
redis_1          | 1:M 08 Sep 2023 08:45:54.217 * Running mode=standalone, port=6379.
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * Server initialized
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * Loading RDB produced by version 7.2.1
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * RDB age 650 seconds
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * RDB memory usage when created 0.85 Mb
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * Done loading RDB, keys loaded: 4, keys expired: 0.
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * DB loaded from disk: 0.000 seconds
redis_1          | 1:M 08 Sep 2023 08:45:54.218 * Ready to accept connections tcp
migrations_1     | Running migrations service
migrations_1     | Operations to perform:
migrations_1     |   Apply all migrations: account, actions, admin, auth, authtoken, contenttypes, files, sessions, sites, socialaccount, users
migrations_1     | Running migrations:
migrations_1     |   No migrations to apply.
migrations_1     | Loaddata has already run
migrations_1     | RUNNING COLLECTSTATIC
migrations_1     |
migrations_1     | 0 static files copied to '/home/mediacms.io/mediacms/static', 1465 unmodified.
mediacms_migrations_1 exited with code 0
db_1             | 2023-09-08 09:53:27.206 BST [22] LOG:  checkpoint starting: time
db_1             | 2023-09-08 09:53:27.219 BST [22] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.003 s, sync=0.004 s, total=0.014 s; sync files=2, longest=0.004 s, average=0.002 s; distance=0 kB, estimate=0 kB
redis_1          | 1:M 08 Sep 2023 09:07:28.255 * 100 changes in 300 seconds. Saving...
redis_1          | 1:M 08 Sep 2023 09:07:28.258 * Background saving started by pid 271
redis_1          | 271:C 08 Sep 2023 09:07:28.265 * DB saved on disk
redis_1          | 271:C 08 Sep 2023 09:07:28.266 * Fork CoW for RDB: current 0 MB, peak 0 MB, average 0 MB
redis_1          | 1:M 08 Sep 2023 09:07:28.358 * Background saving terminated with success
redis_1          | 1:M 08 Sep 2023 09:35:47.986 * 100 changes in 300 seconds. Saving...
redis_1          | 1:M 08 Sep 2023 09:35:47.988 * Background saving started by pid 614
redis_1          | 614:C 08 Sep 2023 09:35:47.996 * DB saved on disk
redis_1          | 614:C 08 Sep 2023 09:35:47.996 * Fork CoW for RDB: current 0 MB, peak 0 MB, average 0 MB
redis_1          | 1:M 08 Sep 2023 09:35:48.089 * Background saving terminated with success

Its stucked on this Fork CoW for RDB moment. 
And this is my config: 

version: "3"

services:
  migrations:
    image: mediacms/mediacms:latest
    volumes:
      - ./:/home/mediacms.io/mediacms/
    environment:
      ENABLE_UWSGI: 'no'
      ENABLE_NGINX: 'no'
      ENABLE_CELERY_SHORT: 'no'
      ENABLE_CELERY_LONG: 'no'
      ENABLE_CELERY_BEAT: 'no'
      ADMIN_USER: 'admin'
      ADMIN_EMAIL: 'admin@localhost'
      ADMIN_PASSWORD: 'blabla'
    command: "./deploy/docker/prestart.sh"
    restart: on-failure
    depends_on:
      redis:
        condition: service_healthy
      db:
        condition: service_healthy
  web:
    image: mediacms/mediacms:latest
    deploy:
      replicas: 1
    ports:
      - "8080:8080"
    volumes:
      - ./:/home/mediacms.io/mediacms/
    environment:
      ENABLE_CELERY_BEAT: 'no'
      ENABLE_CELERY_SHORT: 'no'
      ENABLE_CELERY_LONG: 'no'
      ENABLE_MIGRATIONS: 'no'
    depends_on:
      - migrations
  celery_beat:
    image: mediacms/mediacms:latest
    volumes:
      - ./:/home/mediacms.io/mediacms/
    environment:
      ENABLE_UWSGI: 'no'
      ENABLE_NGINX: 'no'
      ENABLE_CELERY_SHORT: 'no'
      ENABLE_CELERY_LONG: 'no'
      ENABLE_MIGRATIONS: 'no'
    depends_on:
      - redis
  celery_worker:
    image: mediacms/mediacms:latest
    deploy:
      replicas: 1
    volumes:
      - ./:/home/mediacms.io/mediacms/
    environment:
      ENABLE_UWSGI: 'no'
      ENABLE_NGINX: 'no'
      ENABLE_CELERY_BEAT: 'no'
      ENABLE_MIGRATIONS: 'no'
    depends_on:
      - migrations
  db:
    image: postgres:15.2-alpine
    volumes:
      - ../postgres_data:/var/lib/postgresql/data/
    restart: always
    environment:
      POSTGRES_USER: mediacms
      POSTGRES_PASSWORD: mediacms
      POSTGRES_DB: mediacms
      TZ: Europe/London
    healthcheck:
      test: [ "CMD-SHELL", "pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}"]
      interval: 10s
      timeout: 5s
      retries: 5
  redis:
    image: "redis:alpine"
    restart: always
    healthcheck:
      test: ["CMD", "redis-cli","ping"]
      interval: 30s
      timeout: 10s
      retries: 3

Thank you for any help :)

**Comment by mgogoulos** [comment]
I'd suggest you try the default docker-compose.yaml without modifications on a new environment as the one that are suggested to use (eg Ubuntu 22) on the docs. Making any modification can have impact on other things and requires debugging. 
Cheers

**Comment by xansone** [comment]
Oh, i have only one server and didnt want to add a new one, I wanted it as the additional features for me and friends :( As i said im already having flarum, team speak, jellyfin :) Sad, but gonna try if i try new server or something.
